lab-matt/gulpfile.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+const nodemon = require('gulp-nodemon');
+
+const paths = ['*.js', 'lib/*.js', 'model/*.js', 'test/*.js', 'route/*.js'];
+
+gulp.task('lint', function(){
+  return gulp.src(paths)
+  .pipe(eslint()).pipe(eslint.format());
+});
+
+gulp.task('test', function(){
+  return gulp.src(__dirname +'/test/model-route-test.js', {read: false})
```

Comment:
Not necessary but you may want to use '/test/*.js' in the future, this way you won't need to hard-code each test file to run as you create more test files in the future :)